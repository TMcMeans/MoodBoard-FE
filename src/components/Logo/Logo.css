.logo {
  color: #DAA520;
  font-family: Megrim;
  font-size: 72px;
  font-weight: 500;
  margin-top: 0;
  padding-top: 20px;
  text-shadow: 2px 4px 4px rgba(255, 255, 255, 0.25);
  -webkit-text-stroke: 0.5px #000000;
}

