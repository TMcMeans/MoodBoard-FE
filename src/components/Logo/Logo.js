import React from 'react';
import './Logo.css';

const Logo = () => {
  return <h1>MoodBoard</h1>;
};

export default Logo;
