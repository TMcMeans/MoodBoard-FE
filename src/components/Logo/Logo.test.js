import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from './Logo';

describe('Logo', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Logo />);
  });
  it('should render a snapshot with correct data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
