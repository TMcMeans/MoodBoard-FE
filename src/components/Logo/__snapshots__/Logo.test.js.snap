// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render a snapshot with correct data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "MoodBoard",
      "className": "logo",
    },
    "ref": null,
    "rendered": "MoodBoard",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "MoodBoard",
        "className": "logo",
      },
      "ref": null,
      "rendered": "MoodBoard",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
